src/pkg/scan/dao/scan/model.go

```diff
@@ -43,3 +43,68 @@ func (r *Report) TableUnique() [][]string {
 		{"digest", "registration_uuid", "mime_type"},
 	}
 }
+
+// VulnerabilityRecord of an individual vulnerability. Identifies an  individual vulnerability item in the scan.
+// Since multiple scanners could be registered with the projects, each scanner
+// would have it's own definition for the same CVE ID. Hence a CVE ID is qualified along
+// with the ID of the scanner that owns the CVE record definition.
+// The scanner ID would be the same as the RegistrationUUID field of Report.
+// Identified by the `cve_id` and `registration_uuid`.
+// Relates to the image using the `digest` and to the report using the `report UUID` field
+type VulnerabilityRecord struct {
+	ID               int64   `orm:"pk;auto;column(id)"`
+	CVEID            string  `orm:"column(cve_id)"`
+	RegistrationUUID string  `orm:"column(registration_uuid)"`
+	Digest           string  `orm:"column(digest)"`
```

Comment:
Is this `Digest` is the digest of the artifact? If it is, I think we don't need to save it in this model. For a special scanner, the vulnerability is unique no matter which artifact it's.

Reply:
I don't think we intended to introduced image layer ID / layer digest in scope of this changes. If that's the artifact's digest it's probably redundant.

Reply:
agree. If we don't save the layer digest in the DB table, let's remove it from this model.

And in the nativeToRelationalSchemaConverter, i found it always saves the artifact digest to this column.

Reply:
Done. removed the Digest and report_uuid columns